e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -22,13 +28,38 @@ BeforeSuite(async (I, caseViewPage, submitApplicationEventPage, enterFamilyManCa
 
 Before(async I => await I.navigateToCaseDetails(caseId));
 
+Scenario('Judiciary creates blank order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, blankOrder);
+});
+
+Scenario('Judiciary creates interim supervision order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, interimSuperVisionOrder);
+});
+
+Scenario('Judiciary creates final supervision order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, finalSuperVisionOrder);
+});
+
+Scenario('Judiciary creates emergency protection order', async (I, caseViewPage, createOrderEventPage) => {
+  await verifyOrderCreation(I, caseViewPage, createOrderEventPage, emergencyProtectionOrder);
+});
+
+Scenario('Judiciary creates interim care order', async (I, caseViewPage, createOrderEventPage) => {
```

Comment:
Are you going to delete those checks? You mentioned you will like to delete them.. Ah in separate PR ok!